### PR TITLE
Documentation pass

### PR DESCRIPTION
**Pull request by tenevdev**
#### Description
I went through the docs while setting up the project and made some edits.

#### Tests
improbadoc

### CODE REVIEW

File: SpatialGDK/Documentation/content/get-started/build-unreal-fork.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ As the GDK extends Unreal Engine's networking capabilities at its core, to use i
 
 ### Unreal Engine EULA
 
-In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, and agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
+In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github). When you are done you will receive an email invite to joing the EpicGames organisation on Github. You need to be a member in order to access the Unreal Engine fork.
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]:
```suggestion
In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github). When you are done you will receive an email invite to join the EpicGames organisation on Github. You need to be a member in order to access the Unreal Engine fork.
```

**Comment by m-samiec** [comment]
Thanks for the changes, looks good. Adding @ElleEss from tech writing.

File: SpatialGDK/Documentation/content/get-started/tutorial.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ In this ThirdPersonShooter project each `TPSCharacter` contains a variable calle
 To resolve this you need to mark the `CurrentHealth` property for replication, just as you would in the native [Unreal Actor replication](https://docs.unrealengine.com/en-us/Resources/ContentExamples/Networking/1_1) workflow. To do this:
 
 1. In your IDE, open `UnrealGDKThirdPersonShooter\Game\Source\ThirdPersonShooter\Characters\TPSCharacter.h`.
-1. Navigate to the declaration of the `CurrentHealth` variable (line 175), and add the UProperty specifiers `ReplicatedUsing = OnRep_CurrentHealth`. The UProperty should now look like this:
+1. Navigate to the declaration of the `CurrentHealth` variable (line 173), and add the UProperty specifiers `ReplicatedUsing = OnRep_CurrentHealth`. The UProperty should now look like this:
```
Comment (girayimprobable) [inline SpatialGDK/Documentation/content/get-started/tutorial.md]: Let's remove the line here, we'll always overlook keeping this current.

File: SpatialGDK/Documentation/content/get-started/build-unreal-fork.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ As the GDK extends Unreal Engine's networking capabilities at its core, to use i
 
 ### Unreal Engine EULA
 
-In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, and agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
+In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github). When you are done you will receive an email invite to join the EpicGames organisation on Github. You need to be a member in order to access the Unreal Engine fork.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]: Addition to "accept the invite" super useful.

These two final sentences:
"When you are done you will receive an email invite to join the EpicGames organisation on Github. You need to be a member in order to access the Unreal Engine fork."
Makes it seem like there are extra steps on top of those already listed. I had to think about whether there were extra steps. I think we can lose these last two sentences?

Comment (ElleEss) [reply SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]: Aah, I see, good call. It's a potential stumbling block. Let's take your idea and be bossier...

"You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github) and make sure you accept the email invitation from EpicGames."

**Comment by tenevdev** [comment]
@ElleEss I've done the suggested edits and am not planning to make any more changes to this PR. I'm not authorised to merge, though.

**Pull request merged**